data/ecosystem/vendors.yaml

```diff
@@ -41,6 +41,13 @@
   contact: 'hello@betterstack.com'
   oss: false
   commercial: true
+- name: Causely
+  distribution: false
+  nativeOTLP: true
+  url: 'https://www.causely.io/blog/unveiling-the-causal-revolution-in-observability/'
```

Comment:
please point to your documentation around OTLP

Reply:
done